Makefile

```diff
@@ -36,3 +36,11 @@ deb: build docs
 	tar -cvzf $(NAME)_$(VERSION).orig.tar.gz $(UPSTREAM)
 	cp -r debian $(UPSTREAM)
 	cd $(UPSTREAM) && debuild -S -sa --lintian-opts --info --display-info --show-overrides
+
+.PHONY: hackerrank
+hackerrank:
+	rm -rf hackerrank && mkdir -p hackerrank
```

Comment:
@dmalan creating into `hackerrank` directory, to avoid confusing with built files or accidentally installing it. How does that sound?